ui/libs/app-loader/src/delegateConstructors.ts

```diff
@@ -104,10 +104,16 @@ export function createOutputAssessmentWidgetDelegate(
      *
      */
     async getLatestAssessment(): Promise<Assessment | undefined> {
-      const assessments : Record<string, Assessment[]> = await sensemakerStore.getAssessmentsForResources({
-        resource_ehs: [resourceEh],
-        dimension_ehs: [dimensionEh],
-      });
+      let assessments: Record<string, Assessment[]>;
+      try {
+        assessments = await sensemakerStore.getAssessmentsForResources({
+          resource_ehs: [resourceEh],
+          dimension_ehs: [dimensionEh],
+        });
+      } catch (error) {
+        console.warn('Could not get assessments for resource - if there are no dimensions configured you may need to handle that in your delegate creation code');
```

Comment:
Let's change this to "Could not retrieve assessments for the specified resources and dimensions."